contrib/images/desmos-env/Dockerfile

```diff
@@ -4,10 +4,6 @@ RUN apt-get update && \
   apt-get -y upgrade && \
   apt-get -y install curl jq file
 
-ARG UID=1000
-ARG GID=1000
-
-USER ${UID}:${GID}
```

Comment:
Remove these lines since running desmos binary must run mkdir to create initial config files, it requires `root` permission to do it.